### ISSUE TITLE
Unable to replace Discord provider's default scope

### ISSUE THREAD

**Issue by trantinan2512** [nuxt-community/auth-module#1191]
### Version
module: `5.0.0-1622309801.87126e3`
nuxt: `2.15.3`

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [x] universal
 - [ ] spa

### Nuxt configuration
```js
auth: {
  strategies: {
    discord: {
      clientId: '...',
      clientSecret: '...',
      scope: ['identify']
    },
  }
}
```

#### What is expected?
Upon redirecting to oauth page, the default scope `['identify', 'email']` should be replaced with just `['identify']` - which only asks the user for their username and avatar.
#### What is actually happening?
The redirected oauth page still asks for both email and username/avatar.
#### Steps to reproduce
* Use Discord provider to redirect user to oauth page.
* Observe the URL: in `scope` param it's `&scope=identify email identify`.
* Conclusion: The custom scope is being appended instead of getting replaced.

## Checklist
* [x]  I have tested with the latest Nuxt version and the issue still occurs
* [x]  I have tested with the latest module version and the issue still occurs
* [x]  I have searched the issue tracker and this issue hasn't been reported yet

**Comment by Intevel** [comment]
Is this bug still present? @ponpondev

**Comment by ponpondev** [comment]
@Intevel yes. Tested with module version `5.0.0-1643023090.140d3d1` also.
